azure-pipelines.yml

```diff
@@ -0,0 +1,169 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+workspace:
+  clean: all
+
+variables:
+  - group: 'Prod Management Resources'
+  - name: buildConfiguration
+    value: 'release'
+  - name: buildPlatform
+    value: 'any cpu'  
+  - name: GitVersion.AssemblySemFileVer
+    value: ''
+
+pool:
+  name: 'Continuous Integration 02 - SSD - 160ACU'
```

Comment:
Can we use DAS - Continuous Integration Agent pool?

Reply:
Testing

Reply:
@nbowes24 This ones for you to comment on post testing on bigger VM sizes :)

Reply:
Updated AKS agent improved build times significantly